pkgs/development/python-modules/gradio/client.nix

```diff
@@ -86,10 +88,11 @@ buildPythonPackage rec {
     gradio' # ensuring we don't propagate this intermediate build
   ];
 
-  # Add a pytest hook skipping tests that access network, marking them as "Expected fail" (xfail).
+  # A pytest hook which xfails FileNotFoundError.
+  # Gradio devs often fail to upload test assets to pypi.
```

Comment:
Can we use `fetchFromGitHub`?

Reply:
now that `buildNpmPackage` is a thing it may be possible, but better left to a separate PR :)